api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -210,6 +215,39 @@ void shouldRecoverFromErrorsToMakePaymentAndSendEmail() throws JsonProcessingExc
         assertThatPaymentEmailWasSent(newCycle);
     }
 
+    @Disabled("HTBHF-2028")
+    @Test
+    void shouldSendNewChildEmailWhenPaymentCycleIncludesBackdatedVouchers() throws JsonProcessingException, NotificationClientException {
+        // setup some claim variables
+        String cardAccountId = UUID.randomUUID().toString();
+        List<LocalDate> sixWeekOld = asList(LocalDate.now().minusWeeks(6));
+        int cardBalanceInPenceBeforeDeposit = 88;
+
+        wiremockManager.stubSuccessfulEligibilityResponse(sixWeekOld);
+        wiremockManager.stubSuccessfulCardBalanceResponse(cardAccountId, cardBalanceInPenceBeforeDeposit);
+        wiremockManager.stubSuccessfulDepositResponse(cardAccountId);
+        stubNotificationEmailResponse();
+
+        Claim claim = createClaimWithPaymentCycleEndingYesterday(cardAccountId, NO_CHILDREN, LocalDate.now().minusWeeks(7));
+
+        invokeAllSchedulers();
+
+        // confirm new payment cycle created with a payment
+        PaymentCycle newCycle = repositoryMediator.getCurrentPaymentCycleForClaim(claim);
+        PaymentCycleVoucherEntitlement expectedVoucherEntitlement =
+                aPaymentCycleVoucherEntitlementWithBackdatedVouchersForYoungestChild(LocalDate.now(), sixWeekOld);
+        assertPaymentCycleIsIsFullyPaid(newCycle, sixWeekOld, cardBalanceInPenceBeforeDeposit, expectedVoucherEntitlement);
+
+        // confirm card service called to make payment
+        Payment payment = newCycle.getPayments().iterator().next();
+        wiremockManager.assertThatGetBalanceRequestMadeForClaim(payment);
+        wiremockManager.assertThatDepositFundsRequestMadeForClaim(payment);
+
+        // confirm notify component invoked with correct email template & personalisation
+        assertThatNewChildEmailWasSent(newCycle);
```

Comment:
Should we have a specific assertion to make sure that the normal payment email is not sent? Would be nice if we could make it that explicit.

Reply:
I guess the fact that this method verifies a single call should be sufficient however.

Reply:
The `verifyNoMoreInteractions` on the next line covers it, I think

Reply:
Yep, I think this and the verify of a single argument should be fine.